signwriting_evaluation/metrics/symbol_distance.py

```diff
@@ -14,45 +14,45 @@ def __init__(self):
         super().__init__("SymbolsDistances")
         self.symbol_classes = {
             'hand_shapes': range(0x100, 0x205),
-            'contact_symbols': range(0x205, 0x221),
-            'movement_paths': range(0x221, 0x2FF),
```

Comment:
why did you remove all of them? even if removed, maybe leave them as commented out.